server/db/calls_sessions_store.go

```diff
@@ -215,3 +215,34 @@ func (s *Store) GetCallSessionsCount(callID string, opts GetCallSessionOpts) (in
 
 	return count, nil
 }
+
+func (s *Store) IsUserInCall(userID, callID string, opts GetCallSessionOpts) (bool, error) {
```

Comment:
What do you think of the performance impact of this approach? I guess we don't really have any other option, and it's not an event that will happen too often, but just curious.

Reply:
@cpoile I'd expect this to be extremely efficient, given that `callID` is a primary key and the sessions table is very small. Plus the query doesn't need to return any structured data. The only further optimization would be an index on `calls_sessions.userid`, but given the premise (small table + filter on call), it's not worth it since a scan will be the best option anyway. To put it in comparison, I think it could be more costly the RPC hook call itself than the DB query.